src/pages/api/admin/loadClassInfo.ts

```diff
@@ -12,6 +13,7 @@ export interface LoadClassInfoResults {
   topics: Topic[]
   studentsNumber: number
   qstemsNumber: number
+  currentTopic: ID | undefined
```

Comment:
could you make this consistent with the one below?
```suggestion
  currentTopic: ID | null
```

Reply:
> could you make this consistent with the one below?

ok updated here https://github.com/kixlab/kuiz/pull/65/commits/ef2284656a7cac045d24b67a9109ee363885b651 :)